src/react/components/common/assetPreview/videoAsset.tsx

```diff
@@ -225,4 +302,66 @@ export class VideoAsset extends React.Component<IVideoAssetProps> {
             this.props.onDeactivated(this.videoPlayer.current.video.video);
         }
     }
+
+    /**
+     * @name - Add Asset Timeline Tags
+     * @description - Draws small lines to show where visited and tagged frames are on
+     * the video line
+     * @member childAssets - Array of child assets in the video
+     * @member videoDuration - Length (in seconds) of the video
+     */
+    private addAssetTimelineTags(childAssets: any[], videoDuration: number) {
+        const innerHtml: string = this.getRenderedAssetTagLinesText(childAssets, videoDuration);
+        if (!this.timelineElement) {
+            // Find the progress holder so we can add the markers to that
+            const progressControls = document.getElementsByClassName("video-react-progress-control");
+            let progressHolderElement: Element = null;
+
+            // Find any that have an editor page content body type, this will be the one we use
+            Array.from(progressControls).forEach( (element) => {
```

Comment:
Any reason we can't do a standard DOM search?  ex) `document.querySelector('.editor-page-content-body')` or `myElement.querySelector('.editor-page-content-body')` if we already have a scoped element?

Reply:
Updated